src/main/drivers/bus_spi.c

```diff
@@ -116,11 +123,33 @@ uint32_t spiTimeoutUserCallback(SPI_TypeDef *instance)
     return spiDevice[device].errorCount;
 }
 
+
 bool spiBusTransfer(const busDevice_t *bus, const uint8_t *txData, uint8_t *rxData, int length)
 {
-    IOLo(bus->busdev_u.spi.csnPin);
-    spiTransfer(bus->busdev_u.spi.instance, txData, rxData, length);
-    IOHi(bus->busdev_u.spi.csnPin);
+
+    #ifdef USE_DMA_SPI_DEVICE
+        (void)(bus);
+        uint32_t timeoutCheck = millis();
+        memcpy(dmaTxBuffer, (uint8_t *)txData, length);
+        dmaSpiTransmitReceive(dmaTxBuffer, dmaRxBuffer, length, 1);
+        while(dmaSpiReadStatus != DMA_SPI_READ_DONE)
+        {
+            //OMG ANDREY SMIRNOV THANKS FOR THE TROLLOLOLOLOL. 
```

Comment:
Please keep personal bitching in your private space.

Reply:
And this, too.
<img width="651" alt="adding_helio_target_by_piiigletthethird_ _pull_request__6135_ _betaflight_betaflight" src="https://user-images.githubusercontent.com/14850998/41529987-b5e9f6d8-7329-11e8-912b-3b2d57a9cb2e.png">